src/template_project/data_management/stats4schools_smoking_template.py

```diff
@@ -0,0 +1,63 @@
+"""Function(s) for cleaning the data set(s)."""
+
+import pandas as pd
+
+
+def clean_smoking_stats4schools(data):
+    """Clean the smoking data set from stats4schools.
+
+    Original source of the data can be found here: https://www.stem.org.uk/rxvt6.
+
+    Args:
+        data (pandas.DataFrame): The (uncleaned) stats4schools smoking data set.
+
+    Returns:
+        pandas.DataFrame: The cleaned data set.
+
+    """
+    clean = pd.DataFrame()
+
+    clean["gender"] = _clean_gender(data["gender"])
+    clean["marital_status"] = _clean_marital_status(data["marital_status"])
+    clean["smoke"] = _clean_smoke(data["smoke"])
```

Comment:
```suggestion
    clean["current_smoker"] = _clean_current_smoker(data["smoke"])
```
We explain below that we made that up. But important to mirror good practice from EPP.